corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -1343,14 +1344,27 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
      * Utility to render question as select2
      * additionalOptions is passed as object to select2 constructor
      */
-    function select2ify(entry, additionalOptions) {
+    function select2ify(entry, additionalOptions, isMulti) {
         var $input = $('#' + entry.entryId);
         $input.select2(_.extend({
             allowClear: true,
             placeholder: entry.placeholderText,
             escapeMarkup: function (m) { return DOMPurify.sanitize(m); },
         }, additionalOptions));
 
+        applySelect2Labelledby(entry.entryId, isMulti);
+    }
+
+    /**
+     * Select2 ignores labelling from original `<select>` element.
+     * Fix that by applying `aria-labelledby` to the element getting keyboard focus.
+     */
+    function applySelect2Labelledby(entryId, isMulti) {
+        var $input = $('#' + entryId);
```

Comment:
I think it is fair to use `const` and `let` in new code.

Reply:
Seems fair to me. https://github.com/dimagi/commcare-hq/pull/34084/commits/3b28124c76f3448b18b59c060bfc3e5561a4593b